src/components/accessories/tabs/RouterTabs.tsx

```diff
@@ -62,11 +93,22 @@ const RouterTabs: FunctionComponent<IProps> = ({ config, defaultRoute }) => {
       <div className="tabs_content"> 
         <Switch>
           {config.map((item, index) => (
-            <PrivateRoute path={url + item.path} key={index}>
-              <div className="panel">
-                {item.content}
-              </div>
-            </PrivateRoute>
+                   
+            !item.submenu
```

Comment:
...and we don't need all this code related to submenu, because we decided to simplify by putting both form and list in the same page, so we don't need (sub)navigation anymore.